### ISSUE TITLE
JAWS doesn't read aria-describedby text on <input> type="text" while the same is read out by NVDA

### ISSUE THREAD

**Issue by ashankggupta** [FreedomScientific/standards-support#839]
### Summary

I have input fields where there are fields to provide the distance of office from home and number of hours worked, where the text "Miles" and "Hours" are visually present after the text field. I am associating the texts "Miles" and "Hours" with the input text fields using aria-describedby but JAWS does not reads out the text, while the same issue cannot be replicated in NVDA.

If yes, how many miles was it? 
input text field
miles

### Expected result
JAWS should read, "If yes, how many miles was it? edit required miles"

### Actual result
JAWS reads , "If yes, how many miles was it? edit required".

### Example
<input type="text" maxlength="4000" id-"textManyMilesYes" class="form-control form-control-sm" aria-required="true" aria-describedby=lblManyMiles" arai-label="">
<span id="lblManyMiles"> miles </span>
<img width="585" alt="image" src="https://github.com/FreedomScientific/standards-support/assets/170301152/5ae12b6f-0876-45e3-9c07-48e74f222c2e">

### Additional Information
The additional info or description for the fields associated using aria-describedby attribute are not read out by JAWS.

#### JAWS version and build number
2023.2302.15

#### Operating System and version
Windows 10

#### Browser and version:
Chrome 124.0.6367.209

**Comment by stevefaulkner** [comment]
tested with chrome and JAWS could not reproduce

test case

**Comment by stevefaulkner** [comment]
@ashankggupta could you provide a usable example of the code you are using?

**Comment by ashankggupta** [comment]
> tested with chrome 124.0.6367.208 (Official Build) (64-bit) and JAWS 2404.2403.3 could not reproduce
> 
> [test case](https://codepen.io/stevef/pen/OJYMewG)

@stevefaulkner - Please check the below code.

<label for="textManyMilesYes"> If yes, how many miles was it? </label>

<input type="text" maxlength="4000" id-"textManyMilesYes" class="form-control form-control-sm" aria-required="true" aria-describedby="lblManyMiles">
<span id="lblManyMiles">miles</span>

**Comment by stevefaulkner** [comment]
@ashankggupta added your code as another [test case](https://codepen.io/stevef/pen/OJYMewG). 

steps to test:

1. use f key or tab to navigate to each text box
2. observe output, for each textbox

### JAWS output:

#### Speech History

[label] **If yes, how many miles was it?** _Edit_
_Required_
_blank_
[description] **miles**
_Type in text._


_Type in text._
[label] **If yes, how many miles was it?** _Edit_
_Required_
_blank_
[description] **miles**
_Type in text._

**Comment by ashankggupta** [comment]
> @ashankggupta added your code as another [test case](https://codepen.io/stevef/pen/OJYMewG).
> 
> steps to test:
> 
> 1. use f key or tab to navigate to each text box
> 2. observe output, for each textbox
> 
> ### JAWS output:
> #### Speech History
> [label] **If yes, how many miles was it?** _Edit_ _Required_ _blank_ [description] **miles** _Type in text._
> 
> _Type in text._ [label] **If yes, how many miles was it?** _Edit_ _Required_ _blank_ [description] **miles** _Type in text._

@stevefaulkner - I have tested the code pen example you provided and JAWS is not the description text for me while navigating the fields using TAB key. However, the description text is being announced using 'F' key.

Is this something known in JAWS 23?

**Comment by stevefaulkner** [comment]
> Is this something known in JAWS 23?

I am unsure, but as it works as expected in 2024, am closing this issue
